bindings/rust/s2n-tls/src/testing.rs

```diff
@@ -120,8 +120,8 @@ struct CertKeyPair {
 impl Default for CertKeyPair {
     fn default() -> Self {
         CertKeyPair {
-            cert: &include_bytes!("../../../../tests/pems/rsa_4096_sha512_client_cert.pem")[..],
-            key: &include_bytes!("../../../../tests/pems/rsa_4096_sha512_client_key.pem")[..],
+            cert: &include_bytes!("testing/pems/rsa_4096_sha512_client_cert.pem")[..],
```

Comment:
I don't think you need to copy these if you include the [manifest dir](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates):

```suggestion
            cert: &include_bytes!(concat!(env!("CARGO_MANIFEST_DIR"), "/testing/pems/rsa_4096_sha512_client_cert.pem"))[..],
```

Reply:
oh wait now i'm seeing the issue... I'm not sure I like including certs in the final crate artifact just for testing... seems like the wrong solution here

Reply:
So instead should we just not do `cargo publish --dry-run --allow-dirty --all-features` in generate.sh, and only specify `--features` other than testing?

Reply:
well we may want to reconsider having the `testing` feature include certs at all, since applications may also set the `testing` feature, which will end up failing to compile.